### PR TITLE
Javascript code cleanup

### PR DESCRIPTION
**Pull request by colemanw**
Overview
----------------------------------------
Cleans up/removes old javascript code.

### CODE REVIEW

**Comment by KarinG** [comment]
Thanks @colemanw - we have an issue with testing that I discovered today. I’ll re-run the jobs once that has been addressed. I ran out of time today!

**Comment by KarinG** [comment]
I just pushed a trivial commit to retrigger the tests.

File: js/webform_civicrm_admin.js [hunk 1]
```diff
@@ -580,18 +580,5 @@ var wfCiviAdmin = (function ($, D) {
     }
   };
 
-  /**
-   * This block uses CiviCRM's jQuery not Drupal's version
-   * TODO: Move more code here! Drupal's version of jQuery is ancient.
```
Comment (colemanw) [inline js/webform_civicrm_admin.js:585]: Glad we didn't ever do that TODO. Easier to get rid of this block completely.

Comment (KarinG) [reply js/webform_civicrm_admin.js:585]: :-) yes phew!

**Comment by KarinG** [comment]
And we are ✅ payment processing unaffected 👍 Merging!

**Pull request merged**